hudi-spark-datasource/hudi-spark/src/main/antlr4/org/apache/hudi/spark/sql/parser/HoodieSqlCommon.g4

```diff
@@ -48,6 +48,7 @@
  statement
     : compactionStatement                                                       #compactionCommand
     | CALL multipartIdentifier   callArgumentList?    #call
+    | (DESC | DESCRIBE) TABLE? (FORMATTED | EXTENDED) tableIdentifier          #describeTableCommand
```

Comment:
High-level question: do we have to add our own parser to intercept `{ DESC | DESCRIBE } TABLE` since Spark SQL already support such statement: https://spark.apache.org/docs/latest/sql-ref-syntax-aux-describe-table.html?

Reply:
I thought we have to, but I was wrong. Thanks!